### PR TITLE
Artefact sorting

### PR DESCRIPTION
**Pull request by joeflack4**
Updates
```
Artefact sorting
- Update: Added sorting to 'mirror signature via oak' script
```

### CODE REVIEW

**Comment by joeflack4** [comment]
@hrshdhgd @souzadevinicius FYI. Small sorting PR. Feel free to look at / review.

File: src/ontology/mondo-ingest.Makefile [hunk 1]
```diff
@@ -331,10 +331,14 @@ reports/mirror_signature-%.tsv: component-download-%.owl $(COMPONENTSDIR)/%.db m
         python3 $(SCRIPTSDIR)/mirror_signature_via_oak.py --db-path $(COMPONENTSDIR)/$*.db --onto-config-path metadata/$*.yml --outpath $@;\
 	 else\
         $(ROBOT) query -i $(TMPDIR)/$<.owl --query ../sparql/classes.sparql $@;\
+        (head -n 1 $@ && tail -n +2 $@ | sort) > $@-temp;\
```
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:334]: FYI: Sorting of `reports/mirror_signature-%.tsv`

File: src/ontology/mondo-ingest.Makefile [hunk 2]
```diff
@@ -331,10 +331,14 @@ reports/mirror_signature-%.tsv: component-download-%.owl $(COMPONENTSDIR)/%.db m
...
+        mv $@-temp $@;\
     fi
 
 reports/component_signature-%.tsv: $(COMPONENTSDIR)/%.owl
 	$(ROBOT) query -i $< --query ../sparql/classes.sparql $@
+	(head -n 1 $@ && tail -n +2 $@ | sort) > $@-temp
```
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:340]: FYI: Sorting of `reports/component_signature-%.tsv`

File: src/scripts/mirror_signature_via_oak.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def mirror_signature_via_oak(db_path: str, onto_config_path: str, outpath: str)
     uris_owned: List[URI] = [converter.expand(curie) for curie in curies_owned]
 
     # Save
-    df = pd.DataFrame({'?term': uris_owned})
+    df = pd.DataFrame({'?term': uris_owned}).sort_values('?term')
```
Comment (joeflack4) [inline src/scripts/mirror_signature_via_oak.py:46]: FYI: Sorting of `reports/mirror_signature-%.tsv` via OAK. This is just used for NCIT right now.

File: src/ontology/mondo-ingest.Makefile (hunk 1 above)
Comment (matentzn) [reply src/ontology/mondo-ingest.Makefile:334]: Cool! Never new you could do that! Nice one.

**Review by matentzn — COMMENTED** [review]
Great, thanks!

**Pull request merged**